src/overrideFetch.ts

```diff
@@ -30,7 +30,7 @@ export function overrideFetch() {
       }
       const rawRequest = new global.Request(input, init);
       const headers = Object.fromEntries(rawRequest.headers.entries());
-      const query = Object.fromEntries(new URL(url).searchParams);
+      const query = Object.fromEntries(new URL(url).searchParams.entries());
```

Comment:
@AmitShwarts thanks for the PR!
According to what I see, this makes no difference in the tests we have (see the code I used below to verify).
Can you add a failing test that is fixed by this change?

```
      const query = Object.fromEntries(new URL(url).searchParams.entries());
      const query2 = Object.fromEntries(new URL(url).searchParams);
      if (!require('lodash/isEqual')(query, query2)) {
        console.log(JSON.stringify(query)) // <-- this won't print anything
      }
```